process-cli/pom.xml

```diff
@@ -18,12 +288,13 @@
     <dependency>
       <groupId>info.picocli</groupId>
       <artifactId>picocli</artifactId>
-      <version>4.7.1</version>
+      <version>4.7.3</version>
     </dependency>
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>sonarlint-rpc-impl</artifactId>
       <version>${project.version}</version>
+      <scope>compile</scope>
```

Comment:
```suggestion
```

Reply:
Good catch, addressed in [79ddfa3](https://github.com/SonarSource/sonarlint-core/pull/757/commits/79ddfa394bd9a8f78543ac932e2eda5e69cdc53b)